karma.conf.js

```diff
@@ -0,0 +1,20 @@
+const { createKarmaConfig, merge } = require('@vaadin/vaadin-component-dev-dependencies/karma-config.js');
+
+module.exports = config => {
+  config.set(
+    merge(createKarmaConfig(), {
+      coverageIstanbulReporter: {
+        thresholds: {
+          global: {
+            statements: 100,
+            branches: 89,
+            functions: 100,
+            lines: 100
```

Comment:
Good point. I wanted to make the order consistent with the console output for convenience, so that it's easy to go through by visually comparing the lines.

Reply:
Changed the order in skeleton PR vaadin/vaadin-element-skeleton@49eba4c6a3e19bbca147a2f3280f29d02a4e0c25